docs/DEVELOPING.md

```diff
@@ -85,6 +85,19 @@ To add a new custom connector, all you have to do is adding a new directory unde
     end
     ```
 
+It's also possible to add a fail-safe mechanism to the method to also tolerate some of the errors that happen when yielding the documents. To do so you can use `yield_with_handling_tolerable_errors` function:
+    ```ruby
+    def yield_documents
+      foobar_client.files do |file|
+        yield_with_handling_tolerable_errors do
+          serialized_document = serialize(file)
+          yield serialized_document
+        end
+      end
+    end
+    ```
```

Comment:
Oh man, I feel silly but I don't understand why it's not formatted properly, do you know what I'm missing here by chance?

Reply:
I think you need 4 preceding spaces on line 88 and 101, instead of 2.

>Alternatively, you could left-shift the codeblock

Then the number in the numbered list won't work anymore

Reply:
Just tried it and it worked, thanks for the help @seanstory and @wangch079!

Reply:
Result after reformatting: 
<img width="1281" alt="image" src="https://user-images.githubusercontent.com/12238374/204810051-3b23793d-a4d7-48fe-963f-c7e4d8a6a749.png">